### PR TITLE
LibWeb: Add Base::apply_presentational_hints call to <symbol> element 

### PR DESCRIPTION
**Pull request by 0GreenClover0**
Fixes a couple SVGs on ex. https://www.jetbrains.com/clion/

| Before | After |
| --- | --- |
| obraz | obraz |

### CODE REVIEW

File: Userland/Libraries/LibWeb/SVG/SVGSymbolElement.cpp [hunk 1]
```diff
@@ -30,6 +30,8 @@ void SVGSymbolElement::initialize(JS::Realm& realm)
 // https://svgwg.org/svg2-draft/struct.html#SymbolNotes
 void SVGSymbolElement::apply_presentational_hints(CSS::StyleProperties& style) const
 {
+    Base::apply_presentational_hints(style);
```
Comment (awesomekling) [inline Userland/Libraries/LibWeb/SVG/SVGSymbolElement.cpp:33]: This is not making properties inheritable (which is a CSS concept), but rather adding a missing call to the `SVGGraphicsElement` base class which ensures that `SVGSymbolElement` supports all the presentation attributes it's supposed to.

Comment (awesomekling) [reply Userland/Libraries/LibWeb/SVG/SVGSymbolElement.cpp:33]: The patch looks great though, just the commit message is misleading :)

Comment (awesomekling) [reply Userland/Libraries/LibWeb/SVG/SVGSymbolElement.cpp:33]: Also, can we add a test for this? If a layout test is not possible, a ref test should still work :)

Comment (0GreenClover0) [reply Userland/Libraries/LibWeb/SVG/SVGSymbolElement.cpp:33]: Is this better?

Comment (0GreenClover0) [reply Userland/Libraries/LibWeb/SVG/SVGSymbolElement.cpp]: I've removed a commit by mistake and the PR has closed itself. Should I make a new one or a reopening is possible?

**Pull request closed without merging**